### ISSUE TITLE
Require those creating a new layer to make a selection in the Record Type field

### ISSUE THREAD

**Issue by HughCraig** [HughCraig/TLCMap#60]
[Re FW Ontology.txt](https://github.com/HughCraig/TLCMap/files/9286362/Re.FW.Ontology.txt)
It would be good to be able to identify layers with Placename as the record type so that they could serve as a third element of the Gazetteer in its narrow sense, alongside the ANPS data and the NCG data. At present, layers created with historical names and Indigenous names and alternative names generally are available for search by ticking the "Layers" option as well as the "Gazetteer" option on the GHAP window, but this also includes all the other layers, losing some precision. One way of ensuring that placename layers are identifiable would be by requiring users when creating a layer to choose one of the options in the Record Type field. This adds a burden on users creating layers, but not a very heavy one. 
The non-extended Gazetteer would then allow users to choose one of the three elements or any combination of them.
The attached is an email exchange between Rui Liu of the ACDEngine, Bill Pascoe and myself on TLCMap ontology, which gave rise to this issue.

**Comment by IanMcCrabb** [comment]
Perhaps we can consider/prioritize this for development within the NCG data sprint

**Comment by BillPascoe** [comment]
This might make people have to stop and think instead of getting that
satisfying result when they try it first time. They might think they have
to study the meanings of these terms or check with the supervisor and make
a very careful choice and so come back tomorrow which they might not do and
we lose the 30s satisfaction and the customer.

**Comment by HughCraig** [comment]
OK, I take the point. How about this: there is no requirement to choose a Record Type, but if creators of a layer do choose “Placename” in the Record Type pull-down then that layer is included in a GHAP search which puts user-contributed placename layers alongside ANPS and NCG as parts of TLCMap to search. This would mean that when we have the AusStage venues list on-boarded, this can be classed as a “Placename” layer and form part of a delimited placename search.

**Comment by IanMcCrabb** [comment]
Have assigned to Hugh prioritize for Maintenance budget

**Comment by BillPascoe** [comment]
I recommend always avoid any extra burden on user. Never let a single thing
stop the very tenous thread of easy sign up. If we require this then there
are reasons to require many other fields too and it's a slippery slope of
onerousness when someone might be giving our app 30 seconds to decide if it
is easy to use or annoying and so not bother.

**Comment by HughCraig** [comment]
I agree we can abandon the idea of requiring this selection of Record Type. However, if users do select a Record Type, it would be good to be able to use this in searches via a filter. It would be advantageous, for instance, to restrict a search to layers which have been identified as being of the Record Type 'placename' rather than search all layers. This could be an additional dropdown in the search UI, offering all of the Record Types as options with a check box with each.

**Comment by BillPascoe** [comment]
Yes, addition of Record Type and Keyword to the filter options is a high priority. I'm pretty sure there is a job in for that already.
